js/okex.js

```diff
@@ -35,6 +35,8 @@ module.exports = class okex extends okcoinusd {
                 'YOYO': 'YOYOW',
             },
         });
+        result.api.web.get.push('spot/markets/tickers');
```

Comment:
This won't work with python and php, unfortunately, as they both treat assoc keys differently from object properties, contrary to js...

Furthermore, this PR has a more deep problem in it – it will add the endpoint to OKEX, but will reference it from OKCoinUSD, so, instead, the implementation for fetchTickers has to be overrided in the OKEX, leaving the one in OKCoinUSD intact...

If you could, please, patch it, it would awesome! Thx!